src/Native/System.Security.Cryptography.Native/pal_cms.h

```diff
@@ -7,14 +7,25 @@
 #include <openssl/cms.h>
 
 /*
+These values should be kept in sync with System.Security.Cryptography.Pkcs.SubjectIdentifierType.cs
+*/
+enum SubjectIdentifierType : int32_t
+{
+    Unknown = 0,                // Use any of the following as appropriate
+    IssuerAndSerialNumber = 1,  // X509IssuerSerial
+    SubjectKeyIdentifier = 2,   // SKI hex string
```

Comment:
The per-member comments here don't really apply.  You're not getting a "hex string" for SKI(D), but a cert/octet string.